csharp-advanced-topics/ActionFuncDelegatesCsharp/ActionFuncDelegatesCsharp/Program.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+
+namespace ActionFuncDelegatesCsharp
+{
+    public delegate string DelegateFullName(string firstName, string lastName);
+    public delegate void DelegatePrintSumOfNumbers(int firstNumber, int secondNumber);
+    public delegate bool DelegateCheckLengthOfString(string exampleString);
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var firstNumber = 5;
+            var secondNumber = 10;
+            var exampleString = "AmazingCodeMaze";
+            var firstName = "Code";
+            var lastName = "Maze";
+
+            UnderstandingDelegates understandingDelegates = new UnderstandingDelegates();
```

Comment:
Using var keyword whenever we're clear what the type is. Apply to other places as well. Var all the way.